uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -832,6 +832,7 @@ public function upgrade_1035() {
 
     $this->_uninstallActivityTypes('CiviCase', [
       'Background Check',
+      'Background_Check',
```

Comment:
Are `Background_Check` and `Background Check` separate activity types?

Reply:
yes they are. 
![upload](https://user-images.githubusercontent.com/1642119/39241481-0d2f0082-4855-11e8-8407-c76e6ac79412.png)

It seems it got repeated on test sites, not sure why it's like that.

Reply:
Yeah, I can see on staging too. There's also an activity type of the same name for CiviTask. Anyway, this is fine - it makes sense now